src/components/auth/LoginForm.tsx

```diff
@@ -28,31 +29,25 @@ const LoginForm: React.FC = () => {
   const navigate = useNavigate();
 
   const handleLogin = () => {
-    if (passwordValidation(password)) {
-      dispatch(loginSuccess({ email }));
-      navigate('/weather');
-      setErrorMessage(null);
-    } else {
+    if (!passwordValidation(password)) {
       setErrorMessage('Invalid password!');
+      return;
     }
-  };
 
-  const handleLogout = () => {
-    dispatch(logoutSucces());
+    dispatch(loginSuccess({ email }));
+    navigate('/weather');
+    setErrorMessage(null);
   };
 
+  // const handleLogout = () => {
```

Comment:
nemaj naviku commitat/pushat zakomentirani dio koda koji nista ne radi.

Komentari se pushaju kada se objasnjava neki dio koda. Ovo makni

Reply:
Taman sam to radio :)